src/runner/index.js

```diff
@@ -520,6 +532,16 @@ export default class Runner extends EventEmitter {
         this.configuration.mergeOptions({ [OPTION_NAMES.reporter]: reporterOptions });
     }
 
+    _addDashBoardAdvertisementIfNeeded () {
+        const reporterOptions = this.configuration.getOption(OPTION_NAMES.reporter);
+
+        if (!reporterOptions)
```

Comment:
We need to show ads if the default `spec` reporter is used. At present, it does not happen.

`testcafe chrome test.js`

Add a test for this case.